### PR TITLE
feat(hippy-vue): add nativeNode attributes info for debugging

### PR DESCRIPTION
**Pull request by JHGOGOGO**
Before submitting a new pull request, please make sure:

- [x] Test cases have been added/updated/passed for the code you will submit.
- [x] Documentation has added or updated.
- [x] Commit message is following the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters.
- [x] Squash the repeat code commits, short patches are welcome.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=869) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=869) before we can accept your contribution.<br/><hr/>**zexiongli** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=869) it.</sub>

**Comment by zoomchan-cxj** [comment]
fix the unit test problem

File: packages/hippy-vue/src/renderer/native/index.js [hunk 1]
```diff
@@ -246,6 +246,29 @@ function parseViewComponent(targetNode, nativeNode, style) {
...
+ * Get target node attributes, use to chrome devTool tag attribute show while debugging
+ * @param {*} targetNode
+ * @returns attributes
+ */
+ function getTargetNodeAttributes(targetNode) {
+  try {
+    const targetNodeAttributes = JSON.parse(JSON.stringify(targetNode.attributes));
+    const attributes = {
+      id: targetNode.id,
+      class: Array.from(targetNode.classList).join(' '),
```
[the remark this answers is not in the document]
Comment (JHGOGOGO) [reply packages/hippy-vue/src/renderer/native/index.js]: has added default value for avoiding error

**Comment by JHGOGOGO** [comment]
> please fix the unit test problem

has improved unit testing

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**